packages/dev/core/src/Physics/physicsEngineComponent.ts

```diff
@@ -79,7 +78,7 @@ Scene.prototype.getPhysicsEngine = function (): Nullable<IPhysicsEngine> {
  * @param plugin defines the physics engine to be used. defaults to CannonJS.
  * @returns a boolean indicating if the physics engine was initialized
  */
-Scene.prototype.enablePhysics = function (gravity: Nullable<Vector3> = null, plugin?: IPhysicsEnginePlugin): boolean {
+Scene.prototype.enablePhysics = function (gravity: Nullable<Vector3> = null, plugin?: any): boolean {

```

Comment:
Shouldn't it be `IPhysicsEnginePluginV1 | IPhysicsEnginePluginV2`?

Reply:
Fixed!